src/plugins/mer/mersdkdetailswidget.cpp

```diff
@@ -139,6 +146,26 @@ void MerSdkDetailsWidget::setSdk(const MerSdk *sdk)
     }
 
     m_ui->userNameLabelText->setText(sdk->userName());
+
+    auto handler = [this](int hostMemorySize) {
+        int maxMemorySize = m_ui->memorySpinBox->value() > hostMemorySize ? m_ui->memorySpinBox->value() : hostMemorySize;
```

Comment:
There seems to be quite a lot of code duplicated between MerEmulatorDeviceWidget and MerSdkDetailsWidget, and between MerEmulatorDevice and MerSdk at least. We should share the code somehow.

Reply:
Move Gui code in own widget. In MerEmulatorDevice and MerSdk now only get and set functions.